QuantLib/ql/termstructures/volatility/optionlet/optionletstripper.hpp

```diff
@@ -40,6 +40,7 @@ namespace QuantLib {
       public:
         //! \name StrippedOptionletBase interface
         //@{
+        enum Model{Normal, ShiftedLognormal};
```

Comment:
"shifted" as in "using a displacement"? In this case, shouldn't `Normal` be shifted, too? (Or we could just drop the "shifted" from both.)